climada/entity/impact_funcs/test/test_wf.py

```diff
@@ -0,0 +1,72 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Test IFWildfire class.
+"""
+
+import unittest
+import numpy as np
+
+from climada.entity.impact_funcs.wildfire import IFWildfire
+
+THRESH_step = 331
+i_half_check = 523.8
```

Comment:
Why  these  values here and why one capitalized and the other not?

Reply:
arbitrary values for a test case.
I've adjusted the caps